unique-universes/src/exts/minigames.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+import random
+from typing import TYPE_CHECKING
+
+import disnake
+from disnake.ext import commands
+from src.views.shoot import ShootMenu
+
+if TYPE_CHECKING:
+    from src.bot import Universe
+    from src.views.shoot import ShootStats
+
+
+__all__: tuple[str, ...] = (
+    "generate_random_stats",
+    "Minigames",
+)
+
+
+def _generate_fake_mac() -> str:
+    tokens = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+    seq = random.choices(tokens, k=16)  # noqa: S311
+    return ":".join([f"{seq[i]}{seq[i+1]}" for i in range(0, len(seq), 2)])
+
+
+def generate_random_stats(stats: ShootStats, *, skip_health: bool = False) -> None:
+    for attr in stats.__dataclass_fields__:
+        if attr in ("g_acc", "radians_angle", "ammunition", "angle", "total_shots", "hits"):
+            continue
+
+        if attr in ("enemy_position",):
+            x = int(stats.calculate_shot_range())
+            setattr(stats, attr, random.randint(x, x * 2 if x != 0 else 10))  # noqa: S311
+            continue
+
+        if attr in ("enemy_health",) and skip_health:
+            setattr(stats, attr, random.randint(1, 20))  # noqa: S311
+            continue
+
+        setattr(stats, attr, int(random.random() * 100))  # noqa: S311
+
+
+class Minigames(commands.Cog):
+    def __init__(self, bot: Universe) -> None:
+        self.bot = bot
+
+    # disnake typing skill issue
+    @commands.slash_command()  # type: ignore[reportUnknownMemberType]
+    async def shoot(self, inter: disnake.GuildCommandInteraction) -> None:
+        """Run an info overloaded shoot minigame."""
+        player = await self.bot.database.fetch_player(inter.author.id)
+        view = ShootMenu(inter.author, player)
+        generate_random_stats(view.stats)
+        embed = disnake.Embed(title="Shoot minigame", description="\n".join(["." * 10] * 5))
+        embed.add_field("Planet acceleration", f"{round(view.stats.g_acc, 2)} m/s^2")
+        embed.add_field("Position", f"{view.stats.position}")
```

Comment:
```suggestion
        embed.add_field("Position", str(view.stats.position))
```
Here and everywhere else f-strings are used only for casting to string